glide.yaml

```diff
@@ -19,7 +17,7 @@ import:
 - package: github.com/tendermint/go-merkle
   version: develop
 - package: github.com/tendermint/go-p2p
-  version: develop
+  version: viper
```

Comment:
hmm yeah, I've had problems using `glide mirror` I'll give it another shot next time